### ISSUE TITLE
Typo: rename desgeega -> dessgeega

### ISSUE THREAD

**Issue by H-A-M-G-E-R** [PJBoy/lua#2]
I can't contact you on discord because it's gotten a lot worse in 2024: use revolt instead: https://revolt.chat/

**Comment by PJBoy** [comment]
I wasn't sure which spelling I wanted to use for a bunch of these enemies. In the end, I decided to generally follow the MDb bestiary

**Comment by H-A-M-G-E-R** [comment]
internal name and metroid wiki supports the "dessgeega" spelling

**Comment by PJBoy** [comment]
Yeah there's different spellings from different sources. The Metroid wiki is more-or-less just ChozoBoy's opinion https://metroid.fandom.com/wiki/Dessgeega?diff=40928&oldid=40927 , the internal names are very often different than common use (ebi/evir, clayd/kraid, etc)

**Comment by H-A-M-G-E-R** [comment]
Is there an official source with one "s" in "desgeega"?

**Comment by PJBoy** [comment]
I don't think there's any particularly official source whatsoever tbh. Even the NEStroid and SM manuals disagree on enemy names

**Comment by H-A-M-G-E-R** [comment]
However their Japanese names are consistant, that's the result of multiple romanizations.

**Comment by PJBoy** [comment]
Difference of preference, won't fix
